wimprates/halo.py

```diff
@@ -84,36 +89,48 @@ def earth_velocity(t):
     v_earth_sun = v_orbit * (e_1 * np.cos(phi) + e_2 * np.sin(phi))
 
     # Velocity of Local Standard of Rest
-    v_lsr = np.array([0, 220, 0]) * nu.km/nu.s
+    v_lsr = np.array([0, v_0/(nu.km/nu.s), 0]) * nu.km/nu.s
     # Solar peculiar velocity
     v_pec = np.array([11, 12, 7]) * nu.km/nu.s
 
     return v_lsr + v_pec + v_earth_sun
 
 
 @export
-def v_earth(t=None):
+@lru_cache(None)
```

Comment:
I don't think we want an unbound cache size (memory leaks if someone tries a wide range of t), so maybe remove the None? Or remove the cache entirely if we go for a reference time (see below)

Reply:
yes, seems good